### ISSUE TITLE
Gallery item counter

### ISSUE THREAD

**Issue by oozrafa** [biati-digital/glightbox#69]
Is it possible, when there is more than 1 item in a gallery, to add a counter in the top-left corner showing the number total of slides, and the index of the current slide?
eg:
1/4
2/4
3/4
4/4
etc...

Thanks!

**Comment by frankie-tech** [comment]
It's not a perfect solution and it can for sure be improved upon, but I came up with this: [codepen](https://codepen.io/azoicx/pen/zYOVgoj?editors=0110).
I'm sure there is a way to do it without a MutationObserver, but I wasn't able to get the glightbox events to work. I'm sure that was user error though. Lemme know what you think! :)

**Comment by biati-digital** [comment]
Hi @frankie-tech thank you for your solution, I did it for a project recently and this was my solution.

```javascript
const lightbox = GLightbox({
        onOpen: () => {
            const nav = `<div class="project-gallery-nav lightbox-nav">
                        <span class="current gallery-current lightbox-current">1</span>
                        <span class="of"></span>
                        <span class="total gallery-total"></span>
                    </div>`;

            _(lightbox.modal).append(nav);
            let lightboxNav = _('.project-gallery-nav');
            lightboxNav.find('.gallery-total').html(lightbox.elements.length);
        },
        afterSlideChange: (prev, next) => {
            let current = parseInt(next.index) + 1;
            _('.lightbox-current').html(current);
        }
    });
```

No need for MutationObserver, your events are not working because you are adding two parameters when initiating Glightbox.

Your code (wrong)

```javascript
const slides = GLightbox('glightbox', {
	onOpen: () => {
		console.log('open');
	},...
```

Only pass the configuration, checkout the readme if you have questions.
```javascript
const slides = GLightbox({
       selector: 'glightbox', // this is the default so no need to add
	onOpen: () => {
		console.log('open');
	},...
```

**Comment by frankie-tech** [comment]
haha, I knew it was some dumb mistake like that. I didn't know all of those other options were available, either.

I'll just add my new answer in case anyone wants to see :) [codepen](https://codepen.io/azoicx/pen/zYOVgoj)

tldr:
```js
const nav = document.createElement('div');
nav.classList.add('gcounter');
nav.dataset.indicator = '/';

const slides = GLightbox({
	onOpen: function() {
		return slides.modal.appendChild(nav);
	},
	afterSlideChange: function(prev, next) {
		nav.dataset.indicator = `${next.index + 1} / ${slides.elements.length}`;
		nav.classList.add('gcounter-added');
	}
});
```

```css
.gcounter::after {
	content: attr(data-indicator);
}
```

**Comment by biati-digital** [comment]
Awesome, I'm gonna close this for now, I'm not sure this will be added to Glightbox (at least for now), as much as I would like the library to be a complete solution I do not have the time to add many functionalities and also many of the feature request can be achieved with the API and a few lines of code as @frankie-tech just showed us.

In the future I'll try to make some time to improve this library with more functions.
